docs/runtimes/micronaut.md

```diff
@@ -0,0 +1,152 @@
+# Using Micronaut with NubesGen
+
+This documentation is for running Micronaut applications with NubesGen. There are two other options that might interest you:
+
+- You can also use [Java with NubesGen](java/), which is similar to Micronaut, but does not use the specific Micronaut application properties.
+- As Micronaut applications can be packaged with Docker, you can also run them as [Docker applications with NubesGen](docker/).
+
+NubesGen supports creating Azure App Service instances and Azure Functions instances, depending on the type of Micronaut application that you wish to deploy.
+
+## Tutorial: running a Micronaut Launch project with NubesGen
+
+This tutorial is similar to the [Java on NubesGen](java/) tutorial, as this simple Micronaut application is also a Java application.
+
+__Prerequisites:__
+
+_Tip: You can go to [https://aka.ms/nubesgen-azure-shell](https://aka.ms/nubesgen-azure-shell) to have those prerequisites installed, and run the script from a Web browser._
+- [Bash](https://fr.wikipedia.org/wiki/Bourne-Again_shell), which is installed by default on most Linux distributions and on Mac OS X. If you're using Windows, one solution is to use [WSL](https://aka.ms/nubesgen-install-wsl).
+- [Azure CLI](https://aka.ms/nubesgen-install-az-cli). To login, use `az login`.
+- (optional) [GitHub CLI](https://cli.github.com/). To login, use `gh auth login`.
+
+__Steps:__
+1. Create a sample Java Web application using [https://launch.micronaut.io/](https://launch.micronaut.io/).
+   ```bash
+   curl --location --request GET 'https://launch.micronaut.io/create/default/com.example.micronaut-sample-app?lang=JAVA&build=GRADLE&test=JUNIT&javaVersion=JDK_11' | tar -xzvf -
+   ```
+2. Create a project on GitHub called `micronaut-sample-app`, and push the generated project to that repository. Change `<your-github-account>` by the name of your GitHub account:
+   ```bash
+   cd micronaut-sample-app
+   git init
+   git add .
+   git commit -m "first commit"
+   git remote add origin https://github.com/<your-github-account>/micronaut-sample-app.git
+   git branch -M main
+   git push -u origin main
+   ```
+3. In the cloned project (`cd micronaut-sample-app`), set up GitOps with NubesGen by running the NubesGen CLI ([more information here](/gitops/gitops-quick-start/)):
+   ```bash
+    ./nubesgen-cli-linux gitops
+    ```
+4. Use the command-line with NubesGen ([more information here](/reference/rest-api/)) to generate a NubesGen configuration:
+   ```bash
+   curl "https://nubesgen.com/demo.tgz?runtime=micronaut_gradle&application=app_service.standard&gitops=true" | tar -xzvf -
```

Comment:
Tell users that if they selected Maven when they created the Micronaut Application in launch they should use a different environment.